### PR TITLE
Feature/add l18 for translate

### PR DESCRIPTION
**Pull request by barantarasNEW**

### CODE REVIEW

**Review by nosovandriy — APPROVED** [review]

**Review by NataliiaRodoman — APPROVED** [review]

**Review by vanyachyzh — APPROVED** [review]

File: client/public/locales/en/translation.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "yourName": "Your name",
+  "password": "Password",
+  "sendInstructions": "Send instructions",
+  "checkEmail": "Check your email. We have sent you some information.",
+  "goToLogin": "Go to login",
+  "welcomeToUseeSvit": "Welcome to UseeSvit.",
+  "somethingWentWrong": "Something went wrong",
+  "logIn": {
+    "title": "Log In",
+    "btn": "LOGIN"
```
Comment (mykhalenych) [inline client/public/locales/en/translation.json]: don't use upperCase

File: client/public/locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "forgotPassword": {
+    "title": "Forgot your password?",
+    "btn": "Forgot password"
+  },
+  "profileMenu": {
+    "profile": "Profile",
+    "myAccount": "My account",
+    "logOut": "Logout"
+  }
+}
```
Comment (mykhalenych) [inline client/public/locales/en/translation.json]: Screenshot 2023-06-28 at 10 44 49
add empty line in the end of the file

File: client/public/locales/uk/translation.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "planYourTrip": "Плануйте свою подорож",
+  "emailAddress": "Ел. пошта",
```
Comment (mykhalenych) [inline client/public/locales/uk/translation.json]:
```suggestion
  "emailAddress": "Електронна пошта",
```

File: client/src/common/i18n.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import i18n from 'i18next';
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import {initReactI18next} from 'react-i18next';
+
+// eslint-disable-next-line prettier/prettier, import/no-named-as-default-member
```
Comment (mykhalenych) [inline client/src/common/i18n.ts]:
```suggestion
```

File: client/src/common/i18n.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import i18n from 'i18next';
```
Comment (mykhalenych) [inline client/src/common/i18n.ts:3]:
```suggestion
// eslint-disable import/no-named-as-default-member
import i18n from 'i18next';
```

File: client/src/index.tsx [hunk 6]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 
+import './common/i18n';
+
```
Comment (mykhalenych) [inline client/src/index.tsx]:
```suggestion
```

File: client/src/services/common.ts [hunk 7]
```diff
@@ -1,5 +1,6 @@
 import axios, {AxiosError} from 'axios';
 import type {AxiosResponse, Method} from 'axios';
```
Comment (mykhalenych) [inline client/src/services/common.ts:1]:
```suggestion
import type {AxiosResponse, Method} from 'axios';

```

File: client/src/services/common.ts [hunk 8]
```diff
@@ -1,5 +1,6 @@
...
+import i18n from '../common/i18n';
```
Comment (mykhalenych) [inline client/src/services/common.ts:4]:
```suggestion
```

**Review by mykhalenych — CHANGES_REQUESTED** [review]

**Review by nosovandriy — APPROVED** [review]

**Review by NataliiaRodoman — APPROVED** [review]

**Review by mykhalenych — APPROVED** [review]

**Pull request merged**